pages/about.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import * as React from "react";
 import { GithubCircleIcon, WebIcon } from "mdi-react";
+const NTERACT_MEMBERS = require("../nteract-members.json");
```

Comment:
Should we make this work in a clean dev mode setup by presenting an empty set of members?

Reply:
It will use a local `GH_TOKEN` during dev, otherwise `fetchMembers` always returns `[]`.

Reply:
Ah that's sweet.